systemd/armv7hf/jessie/entry.sh

```diff
@@ -9,10 +9,17 @@ if [ "$INITSYSTEM" = "on" ]; then
 	echo -e "${GREEN}Systemd init system enabled."
 	env > /etc/docker.env
 
-	echo -e "#!/bin/bash\n exec $@" > /etc/resinApp.cmd
+	cat <<-EOF > /etc/resinApp.cmd
```

Comment:
Can you change `.cmd` to `.sh`, since `.cmd` is a windows thing whereas `.sh` is usually used for bash scripts (or no extension at all is used at times as well)